include/git2/common.h

```diff
@@ -305,6 +306,9 @@ typedef enum {
  *		>
  *		> - `ciphers` is the list of ciphers that are eanbled.
  *
+ *	* opts(GIT_OPT_ENABLE_OFS_DELTA, int enabled)
+ *
+ *		> Enable or disable ofs_delta capability.
```

Comment:
Sorry, I now realize that I was rather misunderstanding the goal.  It's not to _create_ them, it's to _request_ them to be created from the server (or not).  In that case, given the broad name given to this option, I would say either:

1. This single option controls whether we _create_ offset deltas _and_ will override whether we _request_ offset deltas.  If we do this, I think that the documentation should read something like:

    > Enable or disable the use of "offset deltas" when creating packfiles, and the negotiation of them when talking to a remote server.  Offset deltas store a delta base location as an offset into the packfile from the current location, which provides a shorter encoding and thus smaller resultant packfiles. Packfiles containing offset deltas can still be read. This defaults to enabled.

2. Separate options for each of these two scenarios.  In that case, I would change the name of this option to something like `GIT_OPT_ENABLE_REMOTE_OFS_DELTA`, and document it something like:

> Enable or disable requesting "offset deltas" when communicating with a remote.  Offset deltas store a delta base location as an offset into the packfile from the current location, which provides a shorter encoding and thus smaller resultant packfiles.  This defaults to enabled.

Note that if we go with (1), there's no more coding work to do...  it appears that we simply don't support creating packfiles with ofs_deltas?  (I didn't realize this until now, and somebody should sanity check me on it.)